WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -89,6 +93,17 @@ protected WalletViewModel(UiConfig uiConfig, Wallet wallet) : base(wallet)
 
 		public WalletBalanceChartTileViewModel BalanceChartTile { get; }
 
+		public void NavigateAndHighlight(uint256 txid)
+		{
+			Navigate().To(this, NavigationMode.Clear);
+
+			RxApp.MainThreadScheduler.Schedule(async () =>
```

Comment:
Dispatcher.UIThread.Post... no delay

Reply:
It is not working with Post. It is kinda like the DataGrid has to be loaded on the page when the highlight effect executed otherwise there is no effect.

Reply:
lets solve that in future then.